database/migrations/2021_07_19_123858_create_mismatches_table.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Models\ImportMeta;
+
+class CreateMismatchesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('mismatches', function (Blueprint $table) {
+            $table->id();
+            $table->string('statement_guid');
+            $table->string('property_id');
+            $table->binary('wikidata_value'); // Saving as BLOB to enable storing JSON seralizations
+            $table->text('external_value');
+            $table->string('external_url', 1024)->nullable();
```

Comment:
Here, in contrast, the metadata doc does not say anything about a length limit. I agree that 1024 would be pretty long indeed, but generally URLs up to 2048 seem to be fine.

Reply:
That is a good point, I will set it to 2048, and use the url limits from wikidata, which are 1500

Reply:
@Silvan-WMDE done, thank you!